code42.json

```diff
@@ -239,10 +222,13 @@
                     "example_values": [
                         "1"
                     ],
+                    "contains": [
```

Comment:
Plugs in a lot easier if uses the same contains fields as other actions may expect.
There is also a `user name` contains type but it seemed buggy, makes it hard to input raw strings for some reason, requires data to be classified...